octopus-server/src/main/resources/buildServerResources/v2/subpages/viewBuildInformationParameters.jsp

```diff
@@ -37,8 +37,3 @@
   Overwrite Mode:
   <strong><props:displayValue name="${keys.overwriteModePropertyName}" emptyValue="not specified"/></strong>
 </div>
-
```

Comment:
assuming this moved up onto the common page, right?

Reply:
Yeah it did... this is what was causing the silent JSP loading error (excessive boldness) on the settings page